app/nexus.go

```diff
@@ -323,6 +335,7 @@ func NewNexusCommand(getNexusClientFn GetNexusClientFn) (CommandOut, error) {
 								"This will fail if an endpoint with the same name is already registered",
 							Flags: []cli.Flag{
 								endpointNameFlag,
+								endpointDescriptionOptionalFlag,
```

Comment:
do we need to update description field for command?

Reply:
Updated description.